core/src/typ.rs

```diff
@@ -313,12 +41,14 @@ pub enum TypeF<Ty, RRows, ERows, Te> {
 // be boxed. Hence, we don't need to additionally box `RecordRow`.
 
 /// Concrete, recursive definition for an enum row.
-pub type EnumRow = EnumRowF<Box<Type>>;
+#[derive(Clone, PartialEq, Debug)]
+pub struct EnumRow(pub EnumRowF<Box<Type>>);
```

Comment:
Here (and RecordRow just below) is the newtype wrapper to allow us to implement Display.